common/scala/src/main/resources/application.conf

```diff
@@ -101,6 +101,7 @@ whisk {
                 segment-bytes   =  536870912
                 retention-bytes = 1073741824
                 retention-ms    = 3600000
+                max-message-bytes = ${whisk.activation.payload.max}
```

Comment:
The topic used by the invoker to send the activation back to the controller needs to be configured to handle non-default message sizes. Otherwise, all non-default message sizes, ~1MB, are retrieved via DB polling.